go-controller/pkg/cni/cnishim.go

```diff
@@ -222,6 +222,14 @@ func (p *Plugin) CmdCheck(args *skel.CmdArgs) error {
 		return err
 	}
 
+	// Parse previous result.
```

Comment:
should this commit be a fixup and squash into your other commit that added the function?

Reply:
Done